report-viewer/src/router/index.ts

```diff
@@ -0,0 +1,33 @@
+import {createRouter, createWebHistory, RouteRecordRaw} from "vue-router";
+import Overview from "@/views/OverviewView.vue";
+import ComparisonView from "@/views/ComparisonView.vue"
+import FileUpload from "@/views/FileUpload.vue"
+
+const routes: Array<RouteRecordRaw> = [
+  {
+    path: "/",
+    name: "FileUpload",
+    component: FileUpload
+  },
+  {
+    path: "/overview",
+    name: "OverviewView",
+    component: Overview,
+  },
+  {
+    path: "/comparison",
+    name: "ComparisonView",
```

Comment:
Only the `ComparisonView` View has the word `View` in his name. Should be consistent.